ngraph/frontend/tensorflow/src/op/crop_and_resize.cpp

```diff
@@ -0,0 +1,142 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "op_table.hpp"
+#include "openvino/opsets/opset8.hpp"
+
+using namespace std;
+using namespace ov::opset8;
+
+namespace ov {
+namespace frontend {
+namespace tf {
+namespace op {
+
+OutputVector translate_crop_and_resize_op(const NodeContext& node) {
+    /// ng_input: [batch, image_height, image_width, depth]
+    /// ng_boxes: [num_boxes, 4]; each box is a normalized [0.to 1.] co-ordinate
+    /// [y1,
+    /// x1, y2, x2]
+    /// ng_box_ind: [num_boxes]; i-th ng_box_ind refers to the image to crop and
+    /// ranges from 0 to batch
+    /// ng_crop_size: [crop_height, crop_width];
+
+    /// for each box b specified in ng_boxes:
+    ///  1. crop ng_input[ng_box_ind[b]] w/ co-ordinates in ng_boxes
+    ///  2. resize according to method
+
```

Comment:
please check operation type before translation. <--- do we need it at all? for other operations?
please check a number of inputs

Reply:
we have an exact match in op_table, there is no need to check additionally here